DataDefinitions/Properties/StationModels.de.resx

```diff
@@ -165,4 +165,7 @@
     <value>Wissenschaftlicher Außenposten</value>
     <comment>A Scientific Outpost (currently, Talos 1 &amp; Talos 2 in Peregrina)</comment>
   </data>
-</root>
+  <data name="Ocellus" xml:space="preserve">
+    <value>Ocellus Sternenhafen</value>

```

Comment:
CrowdIn will overwrite these unless we repeat these changes there after merging this PR.

Reply:
That's one thing I really wish CrowdIn did differently. That said, all of the information is here and we can repeat the changes on CrowdIn as soon as we merge.

Reply:
I think it is proper for CrowdIn to retain "server authority" over all translated text. It would not be a good world where devs overruled translators without the translators getting a review pass.

Reply:
Especially in inflected languages, it is very common for devs to screw up, and I've seen it happen.